routes/fields.js

```diff
@@ -35,21 +35,29 @@ module.exports = {
     legend: 'fields.delivery-details.legend',
     label: 'fields.delivery-details.label'
   },
+  'same-address': {
+    display: 'inline',
+    options: [
+       {
+        value: 'fields.same-address.options.yes.value',
```

Comment:
Does this actually work? 

I probably wouldn't want to translate radio button *values* as they're data rather than content, and so would be static across content/language changes, but YMMV.